tests/helper/helper_interactive.go

```diff
@@ -0,0 +1,73 @@
+package helper
+
+import (
+	"bytes"
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/Netflix/go-expect"
+	"github.com/hinshun/vt10x"
+	"github.com/kr/pty"
+)
+
+//func RunInteractive(commonVar CommonVar, interVar Interactive) (string, error) {
+func RunInteractive(commonVar CommonVar, command []string, test func(*expect.Console, *bytes.Buffer) (bytes.Buffer, error)) (string, error) {
+
+	// tmpdir, _ := ioutil.TempDir("", "")
```

Comment:
Please remove all commented code before merging

Reply:
Yes, those will be removed in my next commit.

Reply:
removed!